lua/buffertabs/init.lua

```diff
@@ -54,6 +58,10 @@ local function load_buffers(d_buf)
     local bufs = api.nvim_list_bufs()
 
     bufs = vim.tbl_filter(function(buf)
+        if cfg.show_all then
+            return true
+        end
```

Comment:
I also noticed that this behaviour was removed in this [commit](https://github.com/tomiis4/BufferTabs.nvim/commit/de5557831ed00e2a1b88a8143e78510d9d0df87d), I figured it was accidental as the documentation still exists.

I was wondering why that option didn't do anything.

I do think this option is worth having though as it means immediately loading all the buffers into tabs if you open neovim with several files rather than having to visit each before they become tabs.

The default is still false so no change in behaviour, just an option for those who want it :)